### PR TITLE
Reorder execution, pass response-headers to identify_user and identify_company

### PR DESCRIPTION
**Pull request by XinyuLuan**

### CODE REVIEW

File: examples/bottle/hello.py [hunk 1]
```diff
@@ -64,22 +64,28 @@ def check_login(user, psword):
...
+    body = ''
     if check_login(username, password):
-        return "<p>Your login information was correct.</p>"
+        body = "<p>Your login information was correct.</p>"
     else:
-        return "<p>Login failed.</p>"
+        body = "<p>Login failed.</p>"
+
+    headers = {'Content-Type': 'application/html'}
+    return HTTPResponse(status=201, body=body, **headers)
```
Comment (praves77) [inline examples/bottle/hello.py:88]: status is same for failed login too?

File: examples/flask/hello.py [hunk 2]
```diff
@@ -99,126 +181,57 @@ def get_tasks():
...
-    
+
         <p>My first paragraph.</p>
-    
+
         </body>
         </html>
         """
-    return Response(response=html_text, status=200, mimetype='text/html')
+    return Response(response=html_body, status=201, mimetype='text/html')
```
Comment (praves77) [inline examples/flask/hello.py:199]: original status was 200 - new one is 201?

File: examples/flask/hello.py [hunk 3]
```diff
@@ -99,126 +181,57 @@ def get_tasks():
...
+    xml_body = """
         <note>
         <to>Tove</to>
         <from>Jani</from>
         <heading>Reminder</heading>
         <body>Don't forget me this weekend!</body>
         </note>
         """
-    return Response(response=xml_text, status = 200, mimetype='application/xml')
+    return Response(response=xml_body, status=201, mimetype='application/xml')
```
Comment (praves77) [inline examples/flask/hello.py:212]: status code has changed?

**Comment by praves77** [comment]
other than few nitpics, it looks fine.
@dgilling We've removed all unwanted stuffs.  Is there anything you don't want to publish?

**Comment by dgilling** [comment]
If it's ready, go ahead. @praves77 and @XinyuLuan

**Comment by praves77** [comment]
Waiting for test results. Will merge after +ve test results.

File: examples/bottle/hello.py (hunk 1 above)
Comment (XinyuLuan) [reply examples/bottle/hello.py:88]: Changed

File: examples/flask/hello.py (hunk 2 above)
Comment (XinyuLuan) [reply examples/flask/hello.py:199]: changed

File: examples/flask/hello.py (hunk 3 above)
Comment (XinyuLuan) [reply examples/flask/hello.py:212]: changed

File: examples/flask/hello.py [hunk 4]
```diff
@@ -34,7 +34,7 @@ def mask_event(eventmodel):
     return eventmodel
 
 moesif_settings = {
-    'APPLICATION_ID': 'Your Moesif Application Id',
+    'APPLICATION_ID': 'eyJhcHAiOiI3MTA6NzEyIiwidmVyIjoiMi4wIiwib3JnIjoiMjYyOjU0OSIsImlhdCI6MTY1NjYzMzYwMH0.CiU4oW_PuDXhLM8_scQktI_wrqZcu3GAFxUMb0DyQCc',
```
Comment (praves77) [inline examples/flask/hello.py:37]: @XinyuLuan you have checked in app id. Please self review PR after committing :-} - it helps  a lot.

**Review by praves77 — APPROVED** [review]

File: examples/flask/hello.py (hunk 4 above)
Comment (XinyuLuan) [reply examples/flask/hello.py]: Yes, I will be careful next time, sorry

**Pull request merged**